osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/alarm/ClearAlarmRegisterCommandExecutor.java

```diff
@@ -114,6 +126,24 @@ private Optional<AccessResultCode> clearAlarmRegister(
     }
   }
 
+  private Optional<CosemObject> getObject(final DlmsDevice device, final DlmsObjectType objectType)
```

Comment:
This seems to be a fairly generic function. How could we accomodate reuse?

Reply:
Can promote it to the Object Config Service.